crates/ide/src/diagnostics.rs

```diff
@@ -613,7 +613,7 @@ fn main() {
 pub struct Foo { pub a: i32, pub b: i32 }
 "#,
             r#"
-fn {a:42, b: ()} {}
+fn some(, b: ()} {}
 fn items() {}
```

Comment:
Thanks to this macro text expecting already broken output you can see the result of these changes in that the diff became smaller here 😄